packages/core/src/application.ts

```diff
@@ -47,21 +47,22 @@ export class Application extends Context {
       }
     }
 
-    this._bindSequence();
+    this._bindSequence(options && options.sequence);
 
     this.handleHttp = (req: ServerRequest, res: ServerResponse) =>
       this._handleHttpRequest(req, res);
 
     this.bind('logError').to(this._logError.bind(this));
   }
 
-  protected _bindSequence(): void {
+  public sequence(sequence: Constructor<Sequence>) {
+    this._bindSequence(sequence);
+  }
+
+  protected _bindSequence(sequence?: Constructor<Sequence>): void {
     // TODO(bajtos, ritch, superkhau) figure out how to integrate this single
     // sequence with custom sequences contributed by components
-    const sequence = this.options && this.options.sequence ?
-      this.options.sequence :
-      Sequence;
-    this.bind('sequence').toClass(sequence);
+    this.bind('sequence').toClass(sequence || Sequence);
```

Comment:
Also, in order to save us checks whether `options` is defined or not, I think we should add the following line at the top of the constructor:

```ts
if (!options) options = {};
```